### PR TITLE
Physics documentation

### PR DESCRIPTION
**Pull request by jbm950**
Fixed some consistency issues in the documentation of Physics.vector.point. Also included is a bug fix for the documentation of physics.vector.frame.ReferenceFrame. Each of the fixes are explained in more detail in their respective commit messages.

### CODE REVIEW

File: sympy/physics/vector/frame.py [hunk 1]
```diff
@@ -634,17 +633,12 @@ def orientnew(self, newname, rot_type, amounts, rot_order='', variables=None,
...
-        .orient() documentation:\n
-        ========================
-
         """
 
         newframe = self.__class__(newname, variables, indices, latexs)
         newframe.orient(self, rot_type, amounts, rot_order)
         return newframe
 
-    orientnew.__doc__ += orient.__doc__
```
Comment (moorepants) [inline sympy/physics/vector/frame.py]: If you are going to remove this, you should add in a full explanation of all the options to `orientnew` here.

**Comment by jbm950** [comment]
Added the extra documentation to `orientnew` which is closer to what the original author had intended by sharing the examples from `orient`. Don't merge yet though, I found some more things to fix in the physics documentation this morning.

**Comment by jbm950** [comment]
Some questions before I make the other changes. 

One. Why is the latter half of code on this [page](http://docs.sympy.org/latest/modules/physics/mechanics/masses.html) not syntax highlighted and able to be run using SymPy Live? Looking through the raw source files the only difference I could see syntax wise is that the code that is highlighted is preceded by a double colon without a space in the above text. The code without the highlighting and ability to be run has a space:

This runs/highlights
"the only two attributes of the object.::
```
  >>> from sympy.physics.mechanics import Particle, Point
  >>> from sympy import Symbol
  >>> m = Symbol('m')
```
"

and this does not
"the reference frame is created and the kinematics are done. ::
```
  >> from sympy import symbols
  >> from sympy.physics.mechanics import dynamicsymbols, ReferenceFrame
  >> from sympy.physics.mechanics import RigidBody, Particle, Point, outer
```
"

Is the space by the double colon the source of this bug?

Two. Why is time defined as a dynamic variable on this [page](  http://docs.sympy.org/latest/modules/physics/mechanics/examples/lin_pend_nonmin_example.html?)
It doesn't look this this variable is used in the example and I've not in the past seen it required to define a time variable for dynamic variables to work.

**Comment by moorepants** [comment]
> Is the space by the double colon the source of this bug?

Seems likely.

> Why is time defined as a dynamic variable on this page?

This line uses time explicitly:

```python
>>> op_point = {q1: L, q2: 0, q1d: 0, q2d: 0, q1d.diff(t): 0, q2d.diff(t): 0}
```

It isn't a dynamic symbol (i.e. sympy function) but a constant. If you look in the code for `dynamicsymbols` you'll see that a variable for time is explictly defined.

We should have something like:

```python
from sympy.physics.vector import time as t
from sympy.phsyics.mechanics import time as t
```

It would be much better to have the variable for time available like this. But we also need to allow people to set time..so maybe there needs to be a top level function:

```python
from sympy.physics.vector import set_time
```

So that we can set a global time variable.

**Comment by jbm950** [comment]
I missed that line when looking through thanks. I'll go ahead and try removing the spaces and seeing if that works.

**Comment by jbm950** [comment]
Comparing to the source on the [basic operations](http://docs.sympy.org/latest/tutorial/basic_operations.html) page in the tutorial it looks like the colons are not needed at all and are therefore likely the source of the problem.

**Comment by jbm950** [comment]
Should be fixed now.

File: doc/src/modules/physics/mechanics/masses.rst [hunk 2]
```diff
@@ -21,7 +21,7 @@ Particle
 
 Particles are created with the class ``Particle`` in :mod:`mechanics`.
 A ``Particle`` object has an associated point and an associated mass which are
-the only two attributes of the object.::
```
Comment (moorepants) [inline doc/src/modules/physics/mechanics/masses.rst]: Odd, I was under the impression that the double colon signified a code block in restructuredtext. Maybe Sphinx makes some different assumptions?

See here: http://www.sphinx-doc.org/en/stable/rest.html#source-code

Seems like they are required...

**Comment by jbm950** [comment]
[Basic Operations Raw Code](https://raw.githubusercontent.com/sympy/sympy/master/doc/src/tutorial/basic_operations.rst) and [simplification raw code](https://raw.githubusercontent.com/sympy/sympy/master/doc/src/tutorial/simplification.rst) both leave out the colons and they both work with syntax highlighting and SymPy Live... Interesting to note though, looking specifically at the factor subsection in [simplification](http://docs.sympy.org/latest/tutorial/simplification.html) there are three code blocks of which only the first and last work with SymPy Live and none of them have colons. Glancing at that page it looks like any one line code snippet does not load properly. Due to the absence of colons from all of them I'm guessing its probably more to do with  SymPy Live rather than Sphinx.

**Comment by moorepants** [comment]
The SymPy live stuff is a custom addition to our docs. I'd stick to the proper rst syntax and you can raise an issue about the sympy live and syntax highlighting not working.

**Comment by jbm950** [comment]
Reverted colon changes and opened [Issue #96](https://github.com/sympy/sympy-live/issues/96) in the sympy-live repository

File: doc/src/modules/physics/mechanics/advanced.rst [hunk 3]
```diff
@@ -49,12 +49,6 @@ measure numbers, and have unsorted output from the ``mprint``, ``mpprint``, and
 those functions, as the sorting can increase printing time from seconds to
 minutes.
 
-Differentiating
----------------
-Differentiation of very large expressions can take some time in SymPy; it is
-possible for large expressions to take minutes for the derivative to be
-evaluated. This will most commonly come up in linearization.
-
```
Comment (moorepants) [inline doc/src/modules/physics/mechanics/advanced.rst]: What's the reason for removing this?

Comment (jbm950) [inline doc/src/modules/physics/mechanics/advanced.rst]: Because it is already verbatim there and this is a copy paste duplicate. See lines 21-25

**Pull request merged**